### PR TITLE
Added release notes for 10.15.0.207

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/release-10-15-0/applications-10-15-0-bundle/10-15-0-207.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Cockpit</td>
+<td>The "Help and service" widget icons are no longer missing after a hard refresh.</td>
```
Comment (rssag) [inline content/release-10-15-0/applications-10-15-0-bundle/10-15-0-207.md:26]: are these two distinct widget icons? also, if it's UI texts, then it should probably be bold face, so `**Help** and **Service** widget icons`

File: content/release-10-15-0/applications-10-15-0-bundle/10-15-0-207.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+<td>MTM-47547</td>
+</tr>
+
+<tr>
+<td>Cockpit</td>
+<td>The "Create smart rule" option on the datapoint selection list items has been reintroduced for the data explorer.</td>
```
Comment (rssag) [inline content/release-10-15-0/applications-10-15-0-bundle/10-15-0-207.md:32]: bold face here too, probably

File: content/release-10-15-0/applications-10-15-0-bundle/10-15-0-207.md (hunk 1 above)
Comment (BeateRixen) [reply content/release-10-15-0/applications-10-15-0-bundle/10-15-0-207.md:26]: This refers to various icons in the "Help and service" widget. Perhaps better: In the "Help and service" widget, there are no longer missing icons after a hard refresh. ?

File: content/release-10-15-0/applications-10-15-0-bundle/10-15-0-207.md (hunk 2 above)
Comment (BeateRixen) [inline content/release-10-15-0/applications-10-15-0-bundle/10-15-0-207.md:32]:
```suggestion
<td>In the data explorer, the **Create smart rule** option has been re-added to the context menu for datapoint selection list items.</td>

```

**Review by rssag — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Pull request merged**